jwst/tests_nightly/general/pipelines/test_mrs_spec3.py

```diff
@@ -0,0 +1,98 @@
+import os
+import pytest
+from astropy.io import fits as pf
+from jwst.pipeline.calwebb_spec3 import Spec3Pipeline
+
+pytestmark = [
+    pytest.mark.usefixtures('_jail'),
+    pytest.mark.skipif(not pytest.config.getoption('bigdata'),
+                       reason='requires --bigdata')
+]
+
+
+@pytest.mark.skipif(not pytest.config.getoption('--runslow'),
```

Comment:
JWST devs might want to check why this one test takes over 4 hours, and if it cannot be scaled down to perform faster.